LL/senchas_síl_ébir.ttl

```diff
@@ -1088,7 +1088,7 @@
     rel:hasMet <#Patric>.
 
 <#Patric>
-    a foaf:Person;
+    a irishTitles:Noíb;
```

Comment:
He isn't actually called a saint in the text (this is possibly my mistake). I think the clause, "qui omnes sancti & sanctae sunt", refers to the young people who joined him. Should we still be giving him this title?